controller/comments_blackbox_test.go

```diff
@@ -302,6 +301,19 @@ func (s *CommentsSuite) TestShowCommentWithEscapedScriptInjection() {
 	assertComment(s.T(), result.Data, s.testIdentity, "<img src=x onerror=alert('body') />", rendering.SystemMarkupPlainText)
 }
 
+func (s *CommentsSuite) TestShowCommentWithTextAndCodeblock() {
+	// given
+	fxt := tf.NewTestFixture(s.T(), s.DB, tf.CreateWorkItemEnvironment(), tf.WorkItems(1))
+	wiID := fxt.WorkItems[0].ID
+	body := " Hello World ``` \nint x = 1\n ```"
```

Comment:
could you add quotes in the code section, too ? So your test can verify that they won't be "html escaped"

Reply:
Updated in https://github.com/fabric8-services/fabric8-wit/pull/2236/commits/ae43639b94ee30e7391cf55596fe6742e0d99adc